aws_cloudtrail_rules/aws_console_root_login.yml

```diff
@@ -18,6 +18,14 @@ Description: The root account has been logged into.
 Runbook: >
   Investigate the usage of the root account. If this root activity was not authorized, immediately change the root credentials and investigate what actions the root account took.
 Reference: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html
+SummaryAttributes:
+  - eventName
```

Comment:
Are summary attributes based generated on a per rule-id basis? If so, would we the `eventName` be needed here?

Reply:
@rleighton Please correct me if I am wrong!

Summary attributes are generated standard fields that we calculate regardless of what the defined summary attributes are in the YAML files.

That being said, `eventName` would not be appropriate here -- this rule is specific to login events and so the eventName will always yield `ConsoleLogin` thus making this field extraneous and clutter.
Will remove `eventName` from this file.

Reply:
Summary attributes are `declared` per rule and we will display the top 5 unique of each in the UI.